src/opencl-platform/cl_init_platform.cpp

```diff
@@ -290,9 +290,12 @@ bool cl_check_init(void) {
 
             /* select device based on cli arguments */
             string tmpAttrData = attr_data;
-            
-            // always select last device of type GPU
-            device = device_list[dev];
+
+            // always select last device of type GPU            
```

Comment:
Is the comment correct? doesn't it pick the first device now?

Reply:
Indeed, changed.